### ISSUE TITLE
auto install doesn't set up dnsmasq.conf

### ISSUE THREAD

**Issue by DropbearNinja** [pi-hole/pi-hole#284]
On a fresh raspbian jessie format, using automated install, it detects that dnsmasq isn't installed, so installs, then does a check on dnsmasq.conf and incorrectly determines that there's an existing dnsmasq.conf and therefore does nothing to it... 

> ::: Checking apt-get for upgraded packages.... done!
:::
::: Your pi is up to date! Continuing with pi-hole installation...
:::
::: Checking dependencies:
:::    Checking for dnsutils... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H$
:::    Checking for bc... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H$
:::    Checking for toilet... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^$
:::    Checking for figlet... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^$
:::    Checking for dnsmasq... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-$
:::    Checking for lighttpd... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H$
:::    Checking for php5-common... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H$
:::    Checking for php5-cgi... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H$
:::    Checking for php5... Not found! Installing....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\$
:::    Checking for git... already installed!
:::    Checking for curl... already installed!
:::    Checking for unzip... already installed!
:::    Checking for wget... already installed!
:::
::: Stopping services...^H\^H|^H/^H-^H\^H|^H/^H^H\^H|^H/^H-^H\^H done.
:::
::: Checking for existing base files...
:::    Checking /etc/.pihole is a repo... not found!!
:::    Cloning https://github.com/pi-hole/pi-hole.git into /etc/.pihole...^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|$
:::
::: Checking for existing web interface...
:::    Checking /var/www/html/admin is a repo... not found!!
:::    Cloning https://github.com/pi-hole/AdminLTE.git into /var/www/html/admin...^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^$
:::
::: Installing scripts... done.
:::
::: Installing configs...
:::    Existing dnsmasq.conf found... it is not a pi-hole file, leaving alone!
:::    Copying 01-pihole.conf to /etc/dnsmasq.d/01-pihole.conf... done.
:::
::: Creating log file and changing owner to dnsmasq... done!
:::
::: Installing pihole custom index page... done!
:::
::: Installing latest Cron script... done!
:::
::: Preparing to run gravity.sh to refresh hosts...
::: Running gravity.sh
:::
:::
::: sudo will be used.
::: Neutrino emissions detected...
:::
::: No custom adlist file detected, reading from default file... done!
^H\^H.
:::
::: Getting adaway.org list... List updated, transport successful!
::: Getting adblock.gjtech.net list... List updated, transport successful!
::: Getting hosts-file.net list... List updated, transport successful!
::: Getting www.malwaredomainlist.com list... List updated, transport successful!
::: Getting pgl.yoyo.org list... List updated, transport successful!
::: Getting someonewhocares.org list... List updated, transport successful!
::: Getting winhelp2002.mvps.org list... List updated, transport successful!
::: Getting mirror1.malwaredomains.com list... List updated, transport successful!
:::
::: Aggregating list of domains...^H done!
::: Formatting list of domains to remove comments....^H\^H|^H/^H-^H\^H|^H/^H done!
::: 96496 domains being pulled in by gravity...
::: Removing duplicate domains....^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H\^H|^H/^H-^H done!
::: 87380 unique domains trapped in the event horizon.
::: Formatting domains into a HOSTS file...
:::
::: Adding 8 ad list sources to the whitelist... done!
::: Running whitelist script to update HOSTS file....^H\^H|^H/^H-^H 8 domains whitelisted!
::: Running blacklist script to update HOSTS file....^H\^H 0 domains blacklisted!
:::
::: Refresh lists in dnsmasq...^H\^H|^H/^H-^H\^H|^H/^H done!

**Comment by PromoFaux** [comment]
Hey there,

The existing `dnsmasq.conf` it finds is the default one that ships with `dnsmasq`.

We used to overwrite this, but in an effort to allow users to run Pi-hole along with other services on their Pi, we have now moved our config file to `/etc/dnsmasq.d/01-pihole.conf`, bypassing the need to do anything with `dnsmasq.conf` that is in place.

The check is in there to replace our original(pi-hole flavoured) file with the official default file that comes preinstalled with `dnsmasq`.
